### ISSUE TITLE
ERR_INVALID_ARG_TYPE-HELPPP

### ISSUE THREAD

**Issue by ClueeZhuo** [hexojs/hexo#4263]
TypeError [ERR_INVALID_ARG_TYPE]: The "mode" argument must be integer. Received an instance of Object
    at copyFile (fs.js:1890:10)
    at tryCatcher (D:\wwwroot\loft\node_modules\bluebird\js\release\util.js:16:23)
    at ret (eval at makeNodePromisifiedEval (C:\Users\ClueeZhuo\AppData\Roaming\npm\node_modules\hexo-cli\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:13:39)    at D:\wwwroot\loft\node_modules\hexo-fs\lib\fs.js:144:39
    at tryCatcher (D:\wwwroot\loft\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:729:18)
    at Promise._fulfill (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:673:18)
    at Promise._resolveCallback (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:466:57)
    at Promise._settlePromiseFromHandler (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:559:17)
    at Promise._settlePromise (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:729:18)
    at Promise._fulfill (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:673:18)
    at Promise._resolveCallback (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:466:57)
    at Promise._settlePromiseFromHandler (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:559:17)
    at Promise._settlePromise (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:729:18)
    at Promise._fulfill (D:\wwwroot\loft\node_modules\bluebird\js\release\promise.js:673:18)

**Comment by stevenjoezhang** [comment]
Please temporarily use Node.js v13 instead of v14

**Comment by SukkaW** [comment]
The fix is already implemented. Please wait for Hexo releasing a newer version.

**Comment by SukkaW** [comment]
This issue should be considered as duplicated with https://github.com/hexojs/hexo/issues/4263

**Comment by frankli0324** [comment]
came across this issue
issue 4263 is duplicated issue 4263
hmmmmmmm...... interesting

**Comment by silentred** [comment]
@SukkaW  Same error occured with latest hexo version:

hexo version
```
INFO  Validating config
WARN  Deprecated config detected: "external_link" with a Boolean value is deprecated. See https://hexo.io/docs/configuration for more details.
hexo: 5.2.0
hexo-cli: 4.2.0
os: darwin 18.7.0 10.14.6

node: 14.4.0
v8: 8.1.307.31-node.33
uv: 1.37.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.16.0
modules: 83
nghttp2: 1.41.0
napi: 6
llhttp: 2.0.4
openssl: 1.1.1g
cldr: 37.0
icu: 67.1
tz: 2019c
unicode: 13.0
```

**Comment by toyjack** [comment]
same issue on node/14.15.1

fixed by change back to node/12.16.3
